### DISCUSSION TITLE
Tailwind ver3.2.4: styles don't applied at all

### DISCUSSION THREAD

**Discussion by xmarya** [tailwindlabs/tailwindcss#10090] [Help] [answered]
I started learning Tailwind framework by following the "From Zero to Production" series on the official youtube channel but I'm facing two problems:
the first is that when I remove
`    <script src="https://cdn.tailwindcss.com"></script>
` from the head, nothing from the Tailwind classes that I wrote between tags is applied. just makes an effect when I rewrite the script in the head. 

the second problem is that @apply doesn't work at all and Vscode keeps showing me a warning "Unknown at rule @apply". I've read the docs on the site and everything is ok and nothing different.

here are my dev dependencies :
  "devDependencies": {
    "autoprefixer": "^10.4.13",
    "postcss": "^8.4.20",
    "tailwindcss": "^3.2.4",
    "vite": "^4.0.1"
  }

https://github.com/xmarya/tailwindTest.git

**Comment by wongjn** [comment, marked as answer]
> […] when I remove `<script src="https://cdn.tailwindcss.com"></script>` from the head, nothing from the Tailwind classes that I wrote between tags is applied. just makes an effect when I rewrite the script in the head.

The `content` file globs do not sufficiently cover your code that uses Tailwind classes. Consider making this change:
```diff
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,js}"],
+  content: ["index.html"],
   theme: {
     extend: {},
   },
   plugins: [],
 }
 ```

> `@apply` doesn't work at all

This will work once the above configuration change is made.

> Vscode keeps showing me a warning <q>Unknown at rule `@apply`</q>

This is because VSCode does not understand Tailwind-specific syntax. This message can be ignored.

**Reply by xmarya** [reply]
thanks a lot! my code worked perfectly
